Assets/HoloToolkit/Utilities/Scripts/Billboard.cs

```diff
@@ -29,6 +29,7 @@ public class Billboard : MonoBehaviour
         /// The axis about which the object will rotate.
         /// </summary>
         [Tooltip("Specifies the axis about which the object will rotate.")]
+        [SerializeField]
         private PivotAxis pivotAxis = PivotAxis.XY;
```

Comment:
See Comment from: https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/2036#discussion_r185647500

Reply:
This one is private.

Reply:
Then the other should be too.

Reply:
Agreed!

Reply:
I saw you did it already 😉